spec/factories.rb

```diff
@@ -66,6 +66,11 @@
     end
   end
 
+  factory :step_by_step_page_submitted_for_2i, parent: :step_by_step_page_with_steps do
+    status { "submitted_for_2i" }
+    # must pass `review_requester_id` when invoking
+  end
```

Comment:
This can be removed as it's not used in this PR. My apologies 😅

Reply:
Removed

Reply:
Have spoken IRL 👍 I didn't mean the comment, I meant this entire factory, but I can see it's actually being used in the next commit, so we will need to keep it. Can it be moved to the commit where it is used?